examples/DotNetCore/ChatApp/README.md

```diff
@@ -0,0 +1,90 @@
+# Azure App Configuration - .NET ChatApp Sample
+
+An interactive console chat application that integrates with Azure OpenAI services using Azure App Configuration for dynamic AI Configuration management.
+
+## Overview
+
+This .NET console application provides a seamless chat experience with Azure OpenAI, featuring:
+
+- Integration with Azure OpenAI for chat completions
+- Dynamic AI configuration refresh from Azure App Configuration
+- Secure authentication options using API key or Microsoft Entra ID
+
+## Prerequisites
+
+- .NET 8.0 SDK
+- Azure subscription
+- Azure OpenAI service instance
+- Azure App Configuration service instance
+
+## Setup
+
+### Environment Variables
+
+Set the following environment variable:
+
+- `AZURE_APPCONFIGURATION_ENDPOINT`: Endpoint URL of your Azure App Configuration instance
+
+### Azure App Configuration Keys
+
+Configure the following keys in your Azure App Configuration:
+
+#### OpenAI Connection Settings
+
+- `ChatApp:AzureOpenAI:Endpoint` - Your Azure OpenAI endpoint URL
+- `ChatApp:AzureOpenAI:DeploymentName` - Your Azure OpenAI deployment name
+- `ChatApp:AzureOpenAI:ApiKey` - Key Vault reference to the API key for Azure OpenAI (optional)
+
+#### Chat Completion Configuration
+
+- `ChatApp:Completion` - An AI Configuration entry containing the following settings:

```

Comment:
I think the name of this key needs an update?

Reply:
Yes, updated. Thanks!